README.md

```diff
@@ -1 +1,29 @@
-# hw01
+# Biography of Anna Kenig-Ziesler 
+
+![Image of Me](IMG_0909_smaller.jpeg)
```

Comment:
This image looks great! I see in the title that you made the image smaller which I appreciate! It still seems just a little big, but it is a cute photo and doesn't take up the whole page which I appreciate :)

Reply:
Cute image! I was wondering how to make the image smaller because mine was super big which takes up a whole page. I didn't know we can change the size of a picture, and that's something I would like to learn!